tests/components/unifi/test_config_flow.py

```diff
@@ -236,36 +241,39 @@
 
 async def test_option_flow(hass):
     """Test config flow options."""
-    entry = MockConfigEntry(domain=config_flow.DOMAIN, data={}, options=None)
-    hass.config_entries._entries.append(entry)
+    controller = await setup_unifi_integration(hass, wlans_response=WLANS)
 
-    flow = await hass.config_entries.options.async_create_flow(
-        entry.entry_id, context={"source": "test"}, data=None
+    result = await hass.config_entries.options.async_init(
+        controller.config_entry.entry_id
     )
 
-    result = await flow.async_step_init()
-    assert result["type"] == "form"
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
     assert result["step_id"] == "device_tracker"
 
-    result = await flow.async_step_device_tracker(
+    result = await hass.config_entries.options.async_configure(
+        result["flow_id"],
         user_input={
             config_flow.CONF_TRACK_CLIENTS: False,
             config_flow.CONF_TRACK_WIRED_CLIENTS: False,
             config_flow.CONF_TRACK_DEVICES: False,
+            config_flow.CONF_SSID_FILTER: ["SSID 1"],
             config_flow.CONF_DETECTION_TIME: 100,
-        }
+        },
     )
-    assert result["type"] == "form"
+
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
     assert result["step_id"] == "statistics_sensors"
 
-    result = await flow.async_step_statistics_sensors(
-        user_input={config_flow.CONF_ALLOW_BANDWIDTH_SENSORS: True}
+    result = await hass.config_entries.options.async_configure(
+        result["flow_id"], user_input={config_flow.CONF_ALLOW_BANDWIDTH_SENSORS: True}
     )
-    assert result["type"] == "create_entry"
+
+    assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5028567)